### PR TITLE
chore: add bsc and docker commands to readme

### PR DESCRIPTION
**Pull request by forcodedancing**
### Description

Add bsc  and dokcer commands to readme

### Rationale

update readme

### Example

NA

### Changes

Notable changes: 
* readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,14 +53,40 @@ make install-op
...
+# for mainnet
+# export network=bsc
+
+./target/release/bsc-reth node \
+    --datadir=./datadir \
+    --chain=${network} \
+    --http \
+    --http.addr=0.0.0.0 \
+    --http.port=8545 \
+    --http.api="admin, debug, eth, net, trace, txpool, web3, rpc, reth, ots, eth-call-bundle" \
```
Comment (unclezoro) [inline README.md]: what is this `eth-call-bundle` for? I think as an example, just expose necessary API modules, "admin" "debug" seems not suggested by default?

File: README.md [hunk 2]
```diff
@@ -49,14 +53,40 @@ make install-op
 
 ## Run Reth for BSC
 
-Coming soon......
+The command below is for an archive node. To run a full node, simply add the `--full` tag.
+
+```shell
```
Comment (unclezoro) [inline README.md:58]: can we have an easy command to pull up BSC mainnet and opBNB mainnet using docker image?

File: README.md (hunk 1 above)
Comment (forcodedancing) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (forcodedancing) [reply README.md:58]: added

**Review by yutianwu — APPROVED** [review]

**Review by unclezoro — APPROVED** [review]

**Pull request merged**